Add package: Lit-Element Syntax Highlighting

JeremyBernier opened the pull request:
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks, depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass. 
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can trigger @packagecontrol-bot to re-evaluate your pull request
by pushing a commit or closing and reopening your pull request.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"` 
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.
 
You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists, 
why you believe it is different and needed
below this line. -->
This package provides syntax highlighting for Google's lit-element and lit-html libraries. lit-html enables HTML templating within tagged template literal strings, but without syntax highlighting all the developer will see is a string. I did not find any libraries providing syntax highlighting for lit-html, so I created my own.

JeremyBernier commented:
It's been nearly 2 months, is there any intention of reviewing this PR?

FichteFoll commented:
Yes, sorry for the wait. I've been really busy.

> Note: Only the lit-element.sublime-syntax file is valid. The tmLanguage file will not work (only included it because I had to to get the PR approved by the robot). Unless you're using an older version of Sublime Text, this should not matter.

If it's not valid anyway, just change the minimum ST version requirement to 3092. See also [the releant section in the 'Results help' page](https://github.com/packagecontrol/st_package_reviewer/wiki/Package-checks#sublime-syntax-support-has-been-added-in-build-3092-).

Also, the commit that added a license is not included in any tag yet.

JeremyBernier commented:
No worries, thanks for getting back to me.

Just updated the minimum ST version, removed the tmLanguage file, and added the license to the tag. Let me know if I missed anything.
